doc/doc-site/docs/off-chain/rust-sdk.md

```diff
@@ -1,3 +1,255 @@
 # Rust SDK
 
-More docs coming soon!
+The code for the Rust SDK lives in `/econia/src/rust/sdk`. This code provides programmatic access to Econia exchanges, in addition to offering an example to put it all together shown later in this document.
+
+# Structure
+
+- The main struct of the SDK is `EconiaClient`. This allows you to submit transactions, query events and get a view client (`EconiaViewClient`). To create a new `EconiaClient`, you need a node URL, the Econia contract's address, an `aptos_sdk::types::LocalAccount` and you can optionally pass a client config.
```

Comment:
For consistency with other docs pages, how about line breaking after each sentence? This makes it easier to track diffs and to provide feedback comments

For lists you can do

```
- The first item.
  The second sentence of the first item.
- The second item.
```

Reply:
Note that I won't replicate this comment but there are other places throughout the doc that this applies to